app/src/androidTest/java/ru/tinkoff/favouritepersons/utils/PersonViewElement.kt

```diff
@@ -0,0 +1,74 @@
+package ru.tinkoff.favouritepersons.utils
+
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+import io.github.kakaocup.kakao.text.KTextView
+import io.qameta.allure.kotlin.Allure.step
+import org.hamcrest.Matchers
+import ru.tinkoff.favouritepersons.R
+
+class PersonViewElement(private val position: Int) {
+
+    private val rootView = ViewMatchers.isDescendantOfA(
+        Matchers.allOf(
+            ViewMatchers.withParent(ViewMatchers.withId(R.id.rv_person_list)),
+            ViewMatchers.withParentIndex(position)
+        )
+    )
```

Comment:
```suggestion
class PersonViewElement(parent: Matcher<View>) : KRecyclerItem< PersonViewElement >(parent) {
```

и тогда можно работать с recyclerView и ее элементами